test/EndToEnd/tests/GetPackageTest.ps1

```diff
@@ -181,6 +181,8 @@ function Test-GetPackageForProjectReturnsCorrectPackages2 {
 }
 
 function Test-GetPackageForFSharpProjectReturnsCorrectPackages {
+    [SkipTest('https://github.com/NuGet/Home/issues/12835')]
```

Comment:
Is it correct issue? I get 404 when I open it.

Reply:
Good catch~ I fixed it. Thanks!